packages/now-cli/test/dev/fixtures/monorepo-dynamic-paths/blog/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "with-zones-blog",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next dev -p 4000"
```

Comment:
Is the port necessary here?

Reply:
It comes from the next.js sample. It is the same as the `assetPrefix` thing useful if you want to run `next` directly  instead of through vercel... I'll remove it as it is not necessary for the integration test.